Modix.Services/Core/UserTrackingBehavior.cs

```diff
@@ -69,9 +71,21 @@ private Task OnUserJoinedAsync(IGuildUser guildUser)
             => SelfExecuteOnScopedServiceAsync<IUserService>(x => x.TrackUserAsync(guildUser));
 
         private Task OnGuildMemberUpdatedAsync(IGuildUser oldUser, IGuildUser newUser)
-            => SelfExecuteOnScopedServiceAsync<IUserService>(x => x.TrackUserAsync(newUser));
+            => SelfExecuteOnScopedServiceAsync<IUserService>(x =>
+            {
+                if(newUser.Username == null)
+                    Log.Error($"OnGuildMemberUpdatedAsync:\r\n ~ newUser.Id: {newUser.Id}\r\n ~ newUser.Discriminator: {newUser.Discriminator ?? "null"}\r\n ~ newUser.Nickname: {newUser.Nickname ?? "null"}");
```

Comment:
Serilog formats the messages for you, so these can be simplified to: `Log.Error("OnGuildMemberUpdatedAsync:\r\n ~ newUser.Id: {UserId}\r\n ~ newUser.Discriminator: {UserDiscriminator}\r\n ~ newUser.Nickname: {UserNickname}", newUser.Id, newUser.Discriminator, newUser.Nickname);`

Reply:
Well, shit. That's nice.